src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -384,13 +341,27 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
         |      },
         |      "mode": "RW"
         |    }]
-        |  }
-        |}
-      """.stripMargin.getBytes("UTF-8")
-    groupManager.updateApp(any, any, any, any, any) returns Future.successful(plan)
+      """.stripMargin
+    )
 
-    When("The request is processed")
-    val response = appsResource.create(body, false, auth.request)
+    Then("The return code indicates create success")
+    response.getStatus should be(201)
+  }
+
+  test("Creating an app with an external volume without options and DOCKER containerizer should pass validation") {
```

Comment:
I'd rather have this test case fail. If the user wants to avoid specifying a driver then don't declare a persistent volume - use a docker volume w/ host-path set to volume-name